test/AssertAnnotationTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Fliglio\Web;
+
+class AssertAnnotationTest extends \PHPUnit_Framework_TestCase {
+	
+	const CHARS = '0123456789abcdefghijklmnopqrstuvwxyz';
+
+	/**
+	 * Test that annotations are valid
+	 */
+	public function testValidation() {
+
+		// given
+		$id = rand(1,9999);
+		$string = substr(str_shuffle(self::CHARS), 0, 10);
+		$number = rand();
+		$stringLength = substr(str_shuffle(self::CHARS), 0, 10);
+		$integerMinMax = rand(1,10);
+		$expectedAnnotation = new Annotation($id, $string, $number, $stringLength, $integerMinMax);
+
+		// when
+		$expectedAnnotation->validate();
+
+		// then
+		$this->assertTrue(true);
+	}
+
+	/** @expectedException Fliglio\Web\ValidationException */
```

Comment:
would it be fun to use a dataprovider to test that each annotation works?
https://phpunit.readthedocs.io/en/8.5/writing-tests-for-phpunit.html#data-providers